highscore.lua

```diff
@@ -54,11 +55,22 @@ local update_highscore = function()
 end
 
 local timer = 0
+local count = 0
 minetest.register_globalstep(function(dtime)
-   timer = timer + dtime;
-   if timer >= 60 then
-      update_highscore()
-      write_file()
-      timer = 0
-   end
+    timer = timer + dtime;
+    if timer >= 60 then
+        count = count + 1
+        update_highscore()
+        --write_file()
+        timer = 0
+    end
+    if count >= 5 then
+        write_file(xp_redo.highscore)
+    end
```

Comment:
`count` doesn't do anything useful after first write, you probably meant to reset it here to write every 6 minutes?
```suggestion
    if count >= 5 then
        write_file(xp_redo.highscore)
        count = 0
    end
```

Reply:
ups emberassing that I forgot this but yeah I meant that